tests/test-lapp.lua

```diff
@@ -47,16 +47,16 @@ Various flags and option types
 
 check(simple,
     {'-o','in'},
-    {quiet=false,p=false,o='in',input='<file>'})
+    {quiet=false,p=false,o='in',input='<file>', input_name="stdin"})
```

Comment:
It bothers me that these are lowercase `stdin` instead of any of the more *nixy stream name `STDIN`, device file path `/dev/stdin`, or shortcut `-`. Does this have to match the Lua `io.stdin` module or would it make sense to set this to something else?

Reply:
Lapp entails that you use (stdin/stdout) for the name when giving the type:

```lua
-- simple.lua
local args = require ('pl.lapp') [[
Various flags and option types
  <input> (default stdin)  Optional input file parameter, reads from stdin
]]

for k,v in pairs(args) do
    print(k,v)
end
```

So wouldn't it make sense to keep it that way for the user?

Reply:
Begrudgingly, yes I suppose following that president makes more sense in this case. :man_shrugging: